app/pandas.ipynb

```diff
@@ -208,18 +472,82 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 17,
-   "outputs": [],
    "source": [
-    "# Write your code here"
+    "df[\"Average age\"] = df[\"Age\"].mean()\n",
```

Comment:
In Task 4.2, adding 'Average age' as a column with the same value for each row is not meaningful. Consider calculating the average age once and storing it as a separate variable or using it in a different context.